.env.dev

```
@@ -0,0 +1,42 @@
+# All other settings (API keys, model choices, RAG flags) are managed via the Settings page
+
+# Get your SUPABASE_URL from the Data API section of your Supabase project settings -
+# https://supabase.com/dashboard/project/<your project ID>/settings/api
+SUPABASE_URL=https://mlyqeuqlrhrblaqxuqyh.supabase.co
+
+# Get your SUPABASE_SERVICE_KEY from the API Keys section of your Supabase project settings -
+# https://supabase.com/dashboard/project/<your project ID>/settings/api-keys
+# On this page it is called the service_role secret.
+SUPABASE_SERVICE_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Im1seXFldXFscmhyYmxhcXh1cXloIiwicm9sZSI6InNlcnZpY2Vfcm9sZSIsImlhdCI6MTc1NTIwNDMxMywiZXhwIjoyMDcwNzgwMzEzfQ.jMaCEQxjCSXAcwjOHLvOug9zv9dmoINulT0fGfcPv90
+# Optional: Set log level for debugging
+LOGFIRE_TOKEN=pylf_v1_us_NmDBNgp46zFzlms7fTX5jsGCGG2YW2YHcK73BN0SMz4q
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The LOGFIRE_TOKEN key should go before the SUPABASE_SERVICE_KEY key

(UnorderedKey)